### DISCUSSION TITLE
希望能将代码和软件放到Gitee上

### DISCUSSION THREAD

**Discussion by AkarinLiu** [Molunerfinn/PicGo#727] [General]
Github下载速度太慢了，稳定性就是像育碧的土豆服务器那样

**Comment by Molunerfinn** [comment]
19年找Gitee官方提供api接口用于上传release附件，到现在还没有支持。

https://gitee.com/oschina/git-osc/issues/I118JI#note_5618041

gitee什么时候支持相关api，就能在gitee上下。

**Reply by AkarinLiu** [reply]
建议注册一个域名叫getpicgo.org，然后把软件放在Github Pages上面，这样就不会下载慢
